ocs/workload.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+import importlib
+import concurrent.futures
+
+from ocs import constants
+
+
+log = logging.getLogger(__name__)
+
+
+class WorkLoad(object):
+    def __init__(
+            self, name=None, path=None, wrk_load=None,
+            storage_type='fs', pod=None
+    ):
+        """
+        Args:
+            name (str): name for this workload instance (for identifying in a
+                test run)
+            path (str): Mount point OR blk device on the pod where workload
+                should do IO (note: this need not be known at this
+                point in time)
+            wrk_load (str): example fio, mongodb, pgsql etc.
+            storage_type (str): type on which we will be running IOs,
+                if type is 'fs' we will interpret 'path' as mount point else
+                if type is 'block' we will interpret 'path' as a block device
+            pod (Pod): pod on which we want to run this workload
+        """
+        self.name = name
+        self.path = path
+        self.wrk_load = wrk_load
+        self.storage_type = storage_type
+        # Pod on which we will be running IO
+        self.pod = pod
+        self.wrk_load_dir = os.path.join(
+            constants.TEMPLATE_WORKLOAD_DIR, self.wrk_load
+        )
+        try:
+            self.wrk_load_mod = importlib.import_module(
+                self.wrk_load_dir.self.wrk_load
+            )
+        except ModuleNotFoundError as ex:
+            log.error(f"No workload found with name {self.wrk_load}")
+            log.error(ex)
+            raise ModuleNotFoundError
```

Comment:
use just statement `raise` to re-raise last exception, no need to add ModuleNotFoundError

Reply:
Done